### ISSUE TITLE
Extend Saxon jump grades

### ISSUE THREAD

**Issue by birgander2** [theCrag/website#3807]
**What happened?**

Saxon jump grades seem to be supported until grade 5, higher grades are not recognized:

[Example here](https://www.thecrag.com/en/climbing/germany/sachsische-schweiz/bielatal/route/4191038511)

This should be extended until (at least) 6 or (better) 7. As the "Ethics" for the area correctly states "Jumping is an accepted way of reaching a summit. This can range from slightly more than a stepover to completely insane kamikaze operations. The respective grade system (1-easy to 4-hard) is currently extended to 6 or even 7."

**What you expected:**

That a grade 6 jump is parsed in the same way as [easier ones](https://www.thecrag.com/en/climbing/germany/sachsische-schweiz/bielatal/route/230949441)

Additionally, it would be helpful to show the used grading system one mouse over "Saxon and Jumps" in this example. "V 2" means here a grade V route with a grade 2 jump and not a V2 boulder. This important mouse over information is shown only while editing a route, but not in the route listing or route details.

**Comment by scd** [comment]
I have added the extra jumps grades.

@nicHoch maybe you could comment on the potential confusion between boulder V2 and grade with jump V 2

**Comment by nicHoch** [comment]
I do not think that there is a real potential confusion between boulder V2 and grade V with jump 2. We should just remove the Bouldering Vermin V-scale within the Saxon Grade Context as V grades for Boulders are not used in Saxon. 
But also as it is at the moment a Jump grade should be always delimited by a space like (III 2) and not (III2)

**Comment by scd** [comment]
SX context does not have V-scale bouldering.

<img width="755" alt="Screen Shot 2021-02-01 at 4 15 42 pm" src="https://user-images.githubusercontent.com/373058/106417993-644dc100-64a9-11eb-96d7-12f63cfd87e7.png">

@nicHoch I think that the only thing that remains is to check that we are correctly using the slash everywhere. I think it is missing in the color grade in header and stream. @nicHoch  please confirm

**Comment by nicHoch** [comment]
for testing: 
https://www.thecrag.com/en/climbing/germany/sachsische-schweiz/bielatal/route/231004458

missing slashes in @scd 

- [ ] header
- [ ] listview https://www.thecrag.com/en/climbing/germany/sachsische-schweiz/bielatal/area/231002976
- [ ] stream asecnts 
- [ ] stream edits

**Comment by FF-Flo** [comment]
Regarding the original request, saxon jump grades are now extended, for example:
https://www.thecrag.com/en/climbing/germany/sachsische-schweiz/bielatal/area/230948769

So the issue can be closed?

But I would like to see the notation "6/IV" (instead of the actual "IV 6") like it is used in every saxon and czech guidebook and database.
